src/componnts/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { HiArrowSmRight } from 'react-icons/hi';
+import './style.css';
+
+
+function Cards () {
+    const [cards] = useState ([
```

Comment:
As already mentioned, you don't need to use useState here. You can just define the cards as a stateless variable. Or you can define them in another file under state/models, as can be seen in the main branch